controller/lib/hal/hal.h

```diff
@@ -57,6 +60,17 @@ enum class PinMode : uint8_t {
 
 enum class VoltageLevel : uint8_t { HAL_CONSTANT(HIGH), HAL_CONSTANT(LOW) };
 
+enum class AnalogPinId {
```

Comment:
brief comment?  Specifically, I think we're saying one should use `AnalogPinId::Ai` instead of plain `Ai`, everywhere, but I don't think that's so obvious.

Reply:
Done.